NorthstarDLL/util/printcommands.cpp

```diff
@@ -94,7 +105,22 @@ void ConCommand_find(const CCommand& arg)
 	char pTempName[256];
 	char pTempSearchTerm[256];
 
-	for (auto& map : R2::g_pCVar->DumpToMap())
+	ConCommandBase* var;
+	CCVarIteratorInternal* itint = R2::g_pCVar->FactoryInternalIterator();
+	std::unordered_map<std::string, ConCommandBase*> unsortedConvars;
```

Comment:
Since `ConvarSort` is a basic sort, could this become a normal map instead of an unordered one?

Reply:
That entire sort is pointless as std::map already does that. 
Me when writing that code:
![headempty.jpg](https://github.com/R2Northstar/NorthstarLauncher/assets/97146561/fcb9099c-26db-4e70-bc46-cc56296fd79a)

Will fix shortly

Reply:
Resolved in dc94412a9bb16d6ffca2dc3b8d3171ce7e128ccd